clients/fides-js/rollup.config.mjs

```diff
@@ -16,7 +16,7 @@ const GZIP_SIZE_ERROR_KB = 45; // fail build if bundle size exceeds this
 const GZIP_SIZE_WARN_KB = 35; // log a warning if bundle size exceeds this
 
 // TCF
-const GZIP_SIZE_TCF_ERROR_KB = 87;
+const GZIP_SIZE_TCF_ERROR_KB = 90;
```

Comment:
we're not _at_ this max size yet, but we're getting close to it. Seemed appropriate to keep this more general rather than bumping it with every edit.